compilers/concrete-compiler/compiler/include/concretelang/Runtime/context.h

```diff
@@ -42,7 +43,7 @@ typedef struct RuntimeContext {
 
   RuntimeContext() = delete;
   RuntimeContext(ServerKeyset serverKeyset);
-  ~RuntimeContext() {
+  virtual ~RuntimeContext() {
```

Comment:
That's because in the RuntimeContextManager we delete either a RuntimeContext or a derived DistributedRuntimeContext. The default destructor of the derived overrides.